.github/workflows/minimum.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         python-version: [ '3.8', '3.9', '3.10', '3.11', '3.12']
         os: [ubuntu-latest, windows-latest]
         include:
-          - os: macos-latest
+          - os: macos-11
```

Comment:
I don't think we need to change this. It doesn't really matter what kind of a mac these tests run on

Reply:
Not randomly, the minimum `numpy` which is 1.20, doesn't have the `arm` builds or `universal` that is why.

Reply:
Oh ok